Project 13/entity/element-listener.ts

```diff
@@ -0,0 +1,21 @@
+export namespace Project13 {
+    export class ElementListener {
+        private field: HTMLElement;
+        private readonly name: string;
+        private readonly boundCallback: () => void;
+
+        constructor(field: HTMLElement, name: string, boundCallback: () => void) {
+            this.field = field;
+            this.name = name;
+            this.boundCallback = boundCallback;
+        }
+
+        public listen() {
```

Comment:
```suggestion
        public listen(): void {
```
здесь и далее - функции которые ничего не возвращает можно помечать как void. Дело кодстайла делать так или нет: главное единообразие (также как например проставлять public кейворд или нет)